.checkpatch.conf

```diff
@@ -19,6 +19,9 @@
 # Don't expect SPDX tag in the first line of a file
 --ignore SPDX_LICENSE_TAG
 
+# Don't expect Signed-off-by lines in commit messages
+--no-signoff
```

Comment:
@AntonioND If we’re no longer enforcing that commit messages should be ‘signed-off’, should we remove this step from CONTRIBUTING.md?

Reply:
Good point.